@dg0yt 
The remaining `pcre` is caused by wxwidgets using the default toolkit gtk3 in linux, it will look for _gtk+-3.0.pc_ in the system, and then use the other installed libraries in systems including `pcre`.
Then I tried adding `gtk` as a dependency of wxwidgets, but I get an error:
```
CMake Error at ports/wxwidgets/portfile.cmake:28 (message):
  Port freetype must not be installed when building wxwidgets:x64-linux.
```
Seems like this problem is hard to fix at the moment?
Can we ignore this issue for now?

--- a/ports/wxwidgets/portfile.cmake
+++ b/ports/wxwidgets/portfile.cmake
@@ -89,6 +89,7 @@ vcpkg_cmake_configure(
         -DwxUSE_LIBJPEG=sys
         -DwxUSE_LIBPNG=sys
         -DwxUSE_LIBTIFF=sys
+        -DwxUSE_SECRETSTORE=FALSE
         -DwxBUILD_DISABLE_PLATFORM_LIB_DIR=ON
         -DwxUSE_STL=${WXWIDGETS_USE_STL}
         -DwxUSE_STD_CONTAINERS=${WXWIDGETS_USE_STD_CONTAINERS}

--- a/versions/w-/wxwidgets.json
+++ b/versions/w-/wxwidgets.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "68874302dce3aa7f5b3b45ae247eb9f73e7ba9d1",
+      "git-tree": "793b49ce7710b440be0a451354614e282e6fc9fa",
       "version": "3.1.6",
       "port-version": 1
     },

